ethereum/walletBackend.go

```diff
@@ -0,0 +1,60 @@
+package ethereum
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+
+	ethwallet "perun.network/go-perun/backend/ethereum/wallet"
+	"perun.network/go-perun/wallet"
+)
+
+// WalletBackend provides ethereum specific wallet backend functionality.
+type WalletBackend struct {
+	encParams scryptParams
```

Comment:
Embed/typedef instead?

Reply:
In the same context as described in this [comment](https://github.com/direct-state-transfer/dst-go/pull/50#discussion_r443999203), `WalletBackend` type implements `WalletBackend` interface defined in identity package.